tests/components/withings/test_sensor.py

```diff
@@ -398,7 +119,28 @@ async def test_all_entities(
             )
             state_obj = hass.states.get(entity_id)
 
-            async_assert_state_equals(entity_id, state_obj, expected, attribute)
+            if attribute.entity_registry_enabled_default:
```

Comment:
Should we not use the common fixture `entity_registry_enabled_by_default` and make sure all entities are loaded so we test all?

Reply:
Seems same with below test

Reply:
Didn't even know this existed